composer.json

```diff
@@ -16,11 +16,11 @@
     }
   ],
   "require": {
-    "php": "~7.1.3||~7.2.0||~7.3.0",
+    "php": "^7.1",
```

Comment:
@dominik-rohal-solteq fixed php version support, was not working for 7.4

Reply:
ok